fast64_internal/oot/oot_f3d_writer.py

```diff
@@ -556,27 +574,12 @@ def oot_dl_writer_register():
 
     bpy.types.Object.ootDrawLayer = bpy.props.EnumProperty(items=ootEnumDrawLayers, default="Opaque")
 
+    bpy.types.Scene.ootDLExportSettings = bpy.props.PointerProperty(type=OOTDLExportSettings)
+    bpy.types.Scene.ootDLImportSettings = bpy.props.PointerProperty(type=OOTDLImportSettings)
```

Comment:
-> `scene.fast64.oot` if you will

Reply:
I've also moved the export/import settings for oot_skeleton as well.